R/reportCapacity.R

```diff
@@ -43,13 +48,24 @@ reportCapacity <- function(gdx, regionSubsetList = NULL, t = c(seq(2005, 2060, 5
   # read scalars
   sm_c_2_co2 <- as.vector(readGDX(gdx, "sm_c_2_co2"))
 
-
   # data preparation
   ttot <- as.numeric(as.vector(ttot))
-  vm_cap      <- vm_cap[teall2rlf]
-  vm_cap      <- vm_cap[, ttot, ]
+  vm_cap <- vm_cap[teall2rlf]
+  vm_cap <- vm_cap[, ttot, ]
+
   vm_deltaCap <- vm_deltaCap[teall2rlf]
   vm_deltaCap <- vm_deltaCap[, ttot, ]
+
```

Comment:
I would add a comment here to explain the next lines, something like: 

## apply "modifyInvestmentVariables to shift from the model-internal time coverage (deltacap and investment variables for step t represent the average of the years from t-4years to t) to the general convention for the reporting template (all variables represent the average of the years from   t-2.5years to t+2.5years)